### DISCUSSION TITLE
Sample prometheus operator config for remote write into mimir cluster

### DISCUSSION THREAD

**Discussion by velu4689** [grafana/mimir#4838] [Help and support]
Hi everyone,

I have a mimir cluster that is up & running and am able to configure my prometheus server (running on an ec2 instance) to remote write into the mimir cluster. 

Now am trying to run prometheus inside a kubernetes container with remote write option to the same mimir cluster I have setup earlier. However it fails to write with the following config:

Server:
  remote_write:
    - url: http://<loadbalancer_ip>:9009/api/v1/push
      headers:
        X-Scope-OrgID: <>

Pls let me know if it worked for anyone else or kindly share me a working example. Thanks in advance

**Comment by dimitarvdimitrov** [comment]
can you share what the failures were? Maybe error strings you've seen?

**Comment by velu4689** [comment]
The Pod gets into crashloop with the following config under prometheus operator:
remote_write:
  - url: http://<load_balancer_ip1>:9009/api/v1/push
    headers:
      X-Scope-OrgID: lineage-ec2-multi-url-test-alb
  - url: http://<load_balancer_ip2>:9009/api/v1/push
    headers:
      X-Scope-OrgID: lineage-ec2-multi-url-test-clb

**Reply by pracucci** [reply]
> The Pod gets into crashloop with the following config under prometheus operator

Is it Mimir or the prometheus operator to go in crash loop? Can you share the related logs from the crash looping pod?
